examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        args[2][0] = s2pix_detector(args[0][0], args[1][0])
</pre><p>explain what this line does, name args into sensible some stuff , e.g. </p><p> </p><p><code></code><code></p><p>image,data = args[0][0],args[1][0]</p><p>bbox = s2pix_detector(image,data)</p><p>#store result</p><p>args[2][0] = bbox</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>